js/src/toast.js

```diff
@@ -186,6 +187,13 @@ class Toast {
     )
   }
 
+  _clearTimeout() {
+    if (this._timeout) {
```

Comment:
this check isn't necessary because you can do: `clearTimeout(undefined)` or `clearTimeout(null)` it won't throw any error